patches/api/0419-Make-banners-modifiable-with-both-BannerMeta-and-Blo.patch

```diff
@@ -0,0 +1,27 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: roro1506HD <roro1506mcpro@gmail.com>
+Date: Fri, 17 Mar 2023 21:21:14 +0100
+Subject: [PATCH] Make banners modifiable with both BannerMeta and
+ BlockStateMeta
+
+
+diff --git a/src/main/java/org/bukkit/inventory/meta/BannerMeta.java b/src/main/java/org/bukkit/inventory/meta/BannerMeta.java
+index 4739d2ecc26e7e4adc1b297013da98e12fe58783..7e83ca570e51172f33e3733039ca4c4eccfe69b3 100644
+--- a/src/main/java/org/bukkit/inventory/meta/BannerMeta.java
++++ b/src/main/java/org/bukkit/inventory/meta/BannerMeta.java
+@@ -5,9 +5,13 @@ import org.bukkit.DyeColor;
+ import org.bukkit.block.banner.Pattern;
+ import org.jetbrains.annotations.NotNull;
+ import org.jetbrains.annotations.Nullable;
+-
++// Paper start - Make banners also modifiable with BlockStateMeta
++/**
++ * @deprecated use {@link BlockStateMeta}
++ */
++@Deprecated
++// Paper end - Make banners also modifiable with BlockStateMeta
+ public interface BannerMeta extends ItemMeta {
+-
```

Comment:
Fix the diff to remove the two `-` line deletions. For a change like this, only line additions should exist. You can easily do this by just adding the lines back and rebuilding patches.